packages/react-instantsearch-core/src/components/__tests__/Index.js

```diff
@@ -55,7 +55,8 @@ describe('Index', () => {
   it('calls update if indexName prop changes', () => {
     const context = createContext();
 
-    const wrapper = shallow(
+    // componentDidUpdate wasn't called on `shallow`
```

Comment:
That's unexpected though. It was working fine with the previous implementation?

Reply:
Seems to be a regression on Enzyme 3.9.0 https://github.com/airbnb/enzyme/issues/2020

Fixed in Enzyme 3.10.0 https://github.com/airbnb/enzyme/commit/ebdd4e782d25e7ecd1f5d71cc06ee693d8aedf7c

Reply:
Ok then once it's rebased it will work.